src/loraModem.h

```diff
@@ -78,84 +80,84 @@ struct vector {
 // For each frequency SF7-SF12 are used.
 vector freqs [] = 
 {
-	{ 868100000, 125, 7, 12, 868100000, 125, 7, 12},			// Channel 0, 868.1 MHz/125 primary
-	{ 868300000, 125, 7, 12, 868300000, 125, 7, 12},			// Channel 1, 868.3 MHz/125 mandatory and (SF7BW250)
-	{ 868500000, 125, 7, 12, 868500000, 125, 7, 12},			// Channel 2, 868.5 MHz/125 mandatory
-	{ 867100000, 125, 7, 12, 867100000, 125, 7, 12},			// Channel 3, 867.1 MHz/125 Optional
-	{ 867300000, 125, 7, 12, 867300000, 125, 7, 12},			// Channel 4, 867.3 MHz/125 Optional
-	{ 867500000, 125, 7, 12, 867500000, 125, 7, 12}, 			// Channel 5, 867.5 MHz/125 Optional
-	{ 867700000, 125, 7, 12, 867700000, 125, 7, 12},			// Channel 6, 867.7 MHz/125 Optional 
-	{ 867900000, 125, 7, 12, 867900000, 125, 7, 12},			// Channel 7, 867.9 MHz/125 Optional 
-	{ 868800000, 125, 7, 12, 868800000, 125, 7, 12},			// Channel 8, 868.9 MHz/125 FSK Only										
-	{ 0,         0  , 0,  0, 869525000, 125, 9, 9}				// Channel 9, 869.525 MHz/125 for RX2 responses SF9(10%)
-	// TTN defines an additional channel at 869.525 MHz using SF9 for class B. Not used
+  { 868100000, 125, 7, 12, 868100000, 125, 7, 12, "868.100"},     // Channel 0, 868.1 MHz/125 primary
+  { 868300000, 125, 7, 12, 868300000, 125, 7, 12, "868.300"},     // Channel 1, 868.3 MHz/125 mandatory and (SF7BW250)
+  { 868500000, 125, 7, 12, 868500000, 125, 7, 12, "868.800"},     // Channel 2, 868.5 MHz/125 mandatory
```

Comment:
Should be here 868.500?

Reply:
What do you thing has to be 868.500? Which value should be changed?

Reply:
"868.800"?

Reply:
Ah, now I see - typing error! I will correct it - thanks!